bokeh/io/export.py

```diff
@@ -203,11 +207,13 @@ def get_screenshot_as_png(obj: Union[LayoutDOM, Document], *, driver: Optional[W
 
     return Image.open(io.BytesIO(png)).convert("RGBA").crop((0, 0, width, height))
 
-def get_svgs(obj: Union[LayoutDOM, Document], *, driver: Optional[WebDriver] = None, timeout: int = 5,
+def get_svgs(obj: Union[LayoutDOM, Document], *, driver: "Optional[WebDriver]" = None, timeout: int = 5,
```

Comment:
Quoting `Optional` isn't necessary, though correct.

Reply:
I know, I personally prefer an "all or nothing" approach re: string types but I don't have a strong preference here so if you do feel strongly let me know.